docs/creating-studies.md

```diff
@@ -252,11 +252,15 @@ styling.
   `count_`, and otherwise the word `count` should not be used.
 
 #### Metadata tables
-  - Creating a table called `my_study__meta_date` with two `DATE` columns, `min_date`
+  - Create a table called `my_study__meta_date` with two `DATE` columns, `min_date`
   and `max_date`, and populating it with the start and end date of your study, will
   allow other Cumulus tools to detect study date ranges, and otherwise bakes the
   study date range into your SQL for future reference.
-  - Creating a `my_study__meta_version` with one column, `data_package_version`, and
+    - If you are pulling your dates from resources, it's recommended to cap `max_date` to the
+    current time (`LEAST(max_date, CURRENT_DATE)`), since resource data could have typos or
+    be planned events that put `max_date` in the future (which is both inaccurate and may cause
+    parsing issues if the date is too far forward).
```

Comment:
Well this comes back to what the semantic meaning of this table is - like, is it appropriate to pull the planned date of a followup encounter and use that date in this table? Should this table use creation dates of such follow ups instead (if available)?

I imagine usually, this table could likely be hard-coded for manual studies instead of pulled from resources.

Which - that's another thing I suppose. These questions makes a lot of sense for the thing I think we'll be doing with the study builder and having automatic determination of meta_date. But this advice for manual studies in the docs is just that - advice that could be stretched for odd cases like whether or not to include followup times.

Reply:
> I imagine usually, this table could likely be hard-coded for manual studies instead of pulled from resources.

This is explicitly meant to be the actual available data, not the study period. The goal is to say 'what data to i have', not 'what data do i want', to account for cases on the edge where distributed sites might not have performed an ETL operation at exactly the same time. So I'd still like to get @comorbidity to define the desired behavior, and then walk back exceptions based on that.